src/config/bookconfig.rs

```diff
@@ -17,6 +18,8 @@ pub struct BookConfig {
     multilingual: bool,
     indent_spaces: i32,
 
+    playpen_config: PlaypenConfig,
```

Comment:
Historically the `theme` dir was a catch all for all client side  customization outside of `book.toml`. That's why @azerupi is suggesting the move (for consistency sake).

 This might change in the future as you have suggested but most likely we will need a good thought along with major refactoring and split to plugins.